ansible/roles/dev-desktop/tasks/main.yml

```diff
@@ -93,6 +93,16 @@
     - "quotaon -uv /"
   when: user_quota_check.rc == 0
 
+- name: Copy the scripts to manage quotas
```

Comment:
Hm, I feel like I'd rather see these run by Ansible (automatically on apply), rather than needing to login ourselves to update a particular machine.

Happy to see this merged in the meantime though.

Reply:
Oh I didn't even consider running this through Ansible. The benefit would be that we can then fully rely on a single variable to set the quota for both existing and new users. Let me give that a try...

Reply:
Thanks for catching this, @Mark-Simulacrum! Running this through Ansible is actually much simpler and offers much better maintainability. 🙏